tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/DatabaseSteps.cs

```diff
@@ -8,10 +8,12 @@ namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
     public sealed class DatabaseSteps
     {
         private readonly Settings _settings;
+        private readonly ScenarioContext _context;
```

Comment:
Do we need this? Other than being assigned in the constructor it doesn't look like we're using it anywhere.

Reply:
Great spot! I totally forgot to remove that. Thanks!